### PR TITLE
Add tests for Quiz feature

### PR DESCRIPTION
**Pull request by melvinhernandez**
Pivotal Tracker Chore: [#165165358](https://www.pivotaltracker.com/story/show/165165358)

# Problem
There was 0% test coverage for the Quiz feature.

# Solution

- Create feature tests for quiz feature.
- Setup and create rspec tests for Quiz/Response models and controllers.
- Achieving 76% coverage on current project.

## Relevant Files
I rebased with @juanxcerv's other testing branch so only relevant files in this PR are:
- `app/models/quiz.rb`
- `app/models/response.rb`
- `features/quiz.feature`
- `features/step_definitions/web_steps.rb`
- `features/support/env.rb`
- `spec/controllers/quizzes_controller_spec.rb`
- `spec/controllers/response_controller_spec.rb`
- `spec/models/quiz_spec.rb`
- `spec/models/response_spec.rb`

Run test and check coverage by running `rake`

### CODE REVIEW

File: features/step_definitions/web_steps.rb [hunk 1]
```diff
@@ -65,10 +65,11 @@
 end
 
 When /I repeatedly answer questions/ do
```
Comment (Hekaiyo) [inline features/step_definitions/web_steps.rb:67]: This wasn't working on my computer, but maybe it's my setup that's weird (I couldn't get the test in general to work);
potentially wrap this in a `while page.has_button?('submit-response-btn') do `, or recursively call `step "I repeatedly answer questions"` while the button exists, in order to future-proof it against multiple questions?

Comment (melvinhernandez) [reply features/step_definitions/web_steps.rb:67]: Let me try doing the while loop 👍

File: features/step_definitions/web_steps.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  if page_name == "landing page"
+    visit landing_path
+  end
+  if page_name == "home page"
+    visit root_path
+  end
+end
+
+Then /I should see a sign in button/ do
+  expect(find_button("login-button").value).to eq("Placeholder Login")
```
Comment (Hekaiyo) [inline features/step_definitions/web_steps.rb]: Oh yeah; you copied this from Juan's testing branch, right?
https://github.com/eilenshahbaz/LEAD-Center/pull/18/files/1c33da113fb7da020a813508540f3bf9ce7a37a1#r275173303
tl;dr: should probably check existence of button only, not the value of "Placeholder Login"
```suggestion
  expect(page).to have_button('login-button')
```

**Review by Hekaiyo — COMMENTED** [review]
Otherwise looks good!
